src/plugins/data/common/search/search_source/normalize_sort_request.test.ts

```diff
@@ -67,7 +67,7 @@ describe('SearchSource#normalizeSortRequest', function () {
 
   it('should append default sort options', function () {
     const defaultSortOptions = {
-      unmapped_type: 'boolean',
+      unmapped_type: 'boolean' as 'boolean',
```

Comment:
Nit: Export type `FieldSortOptions` and use it here to avoid casting:

```ts
    const defaultSortOptions: FieldSortOptions = {
      unmapped_type: 'boolean',
    };
```